virttest/qemu_devices/qcontainer.py

```diff
@@ -1194,32 +1249,6 @@ def machine_arm64_mmio(cmd=False):
             """
             logging.warn('Support for aarch64 is highly experimental!')
             devices = []
-            if os.path.exists("/usr/share/AAVMF/AAVMF_CODE.fd"):
-                aavmf_code_file = "/usr/share/AAVMF/AAVMF_CODE.fd"
-                aavmf_vars_file = "/usr/share/AAVMF/AAVMF_VARS.fd"
-            else:
-                aavmf_code_file = "/usr/share/edk2/aarch64/QEMU_EFI-pflash.raw"
```

Comment:
According to the description of 5c5f71f73a907bb9d90b31a4cb619a683d254420, looks we have to keep this compat.

Reply:
Actually, the AAVMF path is a soft link, I think we can use the parent path directly to fix this problem.
```
$ ll /usr/share/AAVMF/
total 0
lrwxrwxrwx. 1 root root 42 Aug 10 02:44 AAVMF_CODE.fd -> ../edk2/aarch64/QEMU_EFI-silent-pflash.raw
lrwxrwxrwx. 1 root root 35 Aug 10 02:44 AAVMF_CODE.verbose.fd -> ../edk2/aarch64/QEMU_EFI-pflash.raw
lrwxrwxrwx. 1 root root 40 Aug 10 02:44 AAVMF_VARS.fd -> ../edk2/aarch64/vars-template-pflash.raw
```

Reply:
Maybe not all the os distros provide these symlinks, and I think that is why https://github.com/avocado-framework/avocado-vt/commit/5c5f71f73a907bb9d90b31a4cb619a683d254420 was introduced.

Reply:
Yes, so now I changed it to the absolute path.